### PR TITLE
Travis check for distribution tarball and RPM build

### PR DESCRIPTION
**Pull request by wrrobin**

### CODE REVIEW

**Comment by jdinan** [comment]
Cool!  Rather than adding two rows to the testing matrix, can we do both in the same row?

File: .travis.yml [hunk 1]
```diff
@@ -383,6 +386,23 @@ script:
...
+          cd $SOS_SRC
+          ./configure --with-ofi=$TRAVIS_INSTALL/libfabric/ --enable-pmi-simple
+          make dist
+          rpmbuild -ta ./sos-*.tar.gz --define "configargs --with-ofi=$TRAVIS_INSTALL/libfabric/ --enable-pmi-simple"
+          tar zxvf sos-*.tar.gz
+          cd sos-*
+          ./autogen.sh
+          mkdir build
+          cd build
+          ../configure --with-ofi=$TRAVIS_INSTALL/libfabric/ --enable-pmi-simple
```
Comment (jdinan) [inline .travis.yml]: `../configure --with-ofi=$TRAVIS_INSTALL/libfabric/ $SOS_GLOBAL_BUILD_OPTS $SOS_BUILD_OPTS`

File: .travis.yml [hunk 2]
```diff
@@ -383,6 +386,23 @@ script:
...
+          make check
```
Comment (jdinan) [inline .travis.yml]:
```
make $TRAVIS_PAR_MAKE check TESTS=
make VERBOSE=1 TEST_RUNNER="mpiexec.hydra -np 2" check
```

Comment (jdinan) [reply .travis.yml]: This will shorten the build time.

File: .travis.yml (hunk 1 above)
Comment (wrrobin) [reply .travis.yml]: Oh. I thought those are the only two flags that we want to be tested and similar to what is expected during IFS release. Thanks, will update it.

**Pull request merged**